examples/app-npm-webpack/messages/ru.json

```diff
@@ -0,0 +1,23 @@
+{
+	"ru": {
+		"intro-1": "Используйте Globalize для интернационализиции вашего приложения.",
+		"number-label": "Независимое число",
+		"currency-label": "Независимая валюта",
+		"date-label": "Независимая дата",
+		"relative-time-label": "Независимое относительное время",
```

Comment:
According to the Russian typography standards only first letter of the header should be capitalized